security/pkg/k8s/controller/workloadsecret.go

```diff
@@ -41,10 +41,12 @@ const (
 	// The Istio secret annotation type
 	IstioSecretType = "istio.io/key-and-cert"
 
-	// Label which determines whether Citadel should watch namespace
-	NamespaceManagedLabel = "ca.istio.io/env"
-
-	// If NamespaceOverrideLabel is found, Citadel will respect whatever value is found
+	// NamespaceManagedLabel and NamespaceOverrideLabel contribute to determining whether or not
+	// a given Citadel instance should operate on a namespace. The behavior is as follows:
+	// 1) If NamespaceOverrideLabel exists and is valid, follow what this label tells us
+	// 2) If not, check NamespaceManagedLabel. If the value matches the Citadel instance's NS, then it should be active
+	// 3) If NamespaceManagedLabel nonexistent or invalid, follow enableNamespacesByDefault environment variable
```

Comment:
Add 4) about the semantics of `enableNamespaceBydefault`

Reply:
Added :+1: